python/README.md

```diff
@@ -8,11 +8,11 @@ To use this helper script, add the [New Relic Python agent](https://docs.newreli
 from flask import Flask
 
 # Add the New Relic APM agent.
-import newrelic
+import newrelic.agent
```

Comment:
The documentation at https://github.com/newrelic/newrelic-python-agent#usage indicates that this is the correct way of importing the agent, and it seemed to work for me locally